tests/connector_setup_tests_test.go

```diff
@@ -0,0 +1,129 @@
+package tests
```

Comment:
Please remove '_tests' from this file name.

Reply:
this test for connector_setup_tests, i fink not correct violate the general naming rules

Reply:
this is correct naming as original source file named `connector_setup_tests.go`, so this is correct naming.